x-pack/plugins/ingest/index.ts

```diff
@@ -9,9 +9,9 @@ import { resolve } from 'path';
 
 import { getConfigSchema, initServerWithKibana, KbnServer } from './server/kibana.index';
 
-const APP_ID = 'infra';
+const APP_ID = 'ingest';
 
-export function infra(kibana: any) {
+export function ingest(kibana: any) {
   return new kibana.Plugin({
     id: APP_ID,
     configPrefix: 'xpack.infra',
```

Comment:
```suggestion
    configPrefix: 'xpack.ingest',
```

Do you want that to be ingest?

Reply:
make sense!!! thanks